packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -177,29 +185,41 @@ class _BottomAppBarState extends State<BottomAppBar> {
     final Color color = widget.color ?? babTheme.color ?? defaults.color!;
     final Color surfaceTintColor = widget.surfaceTintColor ?? babTheme.surfaceTintColor ?? defaults.surfaceTintColor!;
     final Color effectiveColor = isMaterial3 ? color : ElevationOverlay.applyOverlay(context, color, elevation);
+    final Color? shadowColor = widget.shadowColor ?? babTheme.shadowColor ?? defaults.shadowColor;
 
     final Widget child = Padding(
       padding: widget.padding ?? babTheme.padding ?? (isMaterial3 ? const EdgeInsets.symmetric(vertical: 12.0, horizontal: 16.0) : EdgeInsets.zero),
       child: widget.child,
     );
 
-    return SizedBox(
-      height: height,
-      child: PhysicalShape(
-        clipper: clipper,
-        elevation: elevation,
-        color: effectiveColor,
-        clipBehavior: widget.clipBehavior,
-        child: Material(
-          key: materialKey,
-          type: isMaterial3 ? MaterialType.canvas : MaterialType.transparency,
-          elevation: elevation,
-          color: isMaterial3 ? effectiveColor : null,
-          surfaceTintColor: surfaceTintColor,
-          child: SafeArea(child: child),
-        ),
-      ),
+    final Material material = Material(
+      key: materialKey,
+      type: isMaterial3 ? MaterialType.canvas : MaterialType.transparency,
+      elevation: elevation,
+      color: isMaterial3 ? effectiveColor : null,
+      surfaceTintColor: surfaceTintColor,
+      shadowColor: shadowColor,
+      child: SafeArea(child: child),
     );
+
+    final PhysicalShape physicalShape = isMaterial3
+        ? PhysicalShape(
+            clipper: clipper,
+            elevation: elevation,
+            shadowColor: shadowColor!,
+            color: effectiveColor,
+            clipBehavior: widget.clipBehavior,
+            child: material,
+          )
+        : PhysicalShape(
+            clipper: clipper,
+            elevation: elevation,
+            color: effectiveColor,
+            clipBehavior: widget.clipBehavior,
+            child: material,
+          );
```

Comment:
```suggestion
    final PhysicalShape physicalShape = PhysicalShape(
            clipper: clipper,
            elevation: elevation,
            shadowColor: isMaterial3 ? shadowColor! : Color(0xFF000000),
            color: effectiveColor,
            clipBehavior: widget.clipBehavior,
            child: material,
     );
```

Reply:
Thanks - though I've made the color fully transparent instead.

Reply:
Back to your suggestion, after testing on M2! It's a required default there.